xml/System.Buffers/IBufferWriter`1.xml

```diff
@@ -64,9 +64,9 @@
         <Parameter Name="sizeHint" Type="System.Int32" />
       </Parameters>
       <Docs>
-        <param name="sizeHint">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="sizeHint">The size of the memory span requested. If 0, requests currently available memory. </param>
+        <summary>Requests a memory span that is at least <paramref name="sizeHint" /> in size if possible.</summary>
```

Comment:
Should be requests a `Memory<T>`, not `memory span`. `Memory<T>` and `Span<T>` are two distinct types.